### PR TITLE
Add int8range type support.

### PR DESCRIPTION
**Pull request by ElectronicRU**
Additionally, turn int4range and int8range into first-class binary types.

### CODE REVIEW

**Comment by ElectronicRU** [comment]
For some reason replication tests fail on OTP 19. May be a transient error, not experienced enough to know.

File: test/epgsql_tests.erl [hunk 1]
```diff
@@ -959,7 +959,18 @@ range_type_test(Module) ->
...
+                      {984655, plus_infinity}, {minus_infinity, plus_infinity}])
+      end,
+      []).
+
+range8_type_test(Module) ->
+    with_min_version(
+      Module,
+      9.2,
+      fun(_C) ->
+          check_type(Module, int8range, "int8range(10, 20)", {10, 20},
```
Comment (seriyps) [inline test/epgsql_tests.erl]: It would be nice to test values greater than int32. Ideally - upper and lower bounds.

**Comment by seriyps** [comment]
@MikhailKalashnikov do you have a idea why replication tests fail here?

**Comment by seriyps** [comment]
Thanks!

**Comment by seriyps** [comment]
It looks like we have some race-condition or timing issue in our replication tests. Will investigate later.

**Comment by ElectronicRU** [comment]
@seriyps looks to me like it might be caused by two replication tests for different Erlang versions coinciding in time.

**Pull request merged**